storage/innobase/include/log0log.h

```diff
@@ -812,9 +812,12 @@ It just need to flush the file header block with current master key.
 @param[in]	iv	encryption iv
 @param[in]	is_boot	if it is for bootstrap
 @param[in]	redo_log_encrypt	encryption type
+@param[in]	version	                key's version (used for KEYRING
+                                        encryption)
```

Comment:
align the 'encryption' to the above line word "key's"

Reply:
done